src/Nest/Modules/SnapshotAndRestore/Repositories/S3Repository.cs

```diff
@@ -125,6 +147,20 @@ public class S3RepositorySettingsDescriptor
 
 		public S3RepositorySettingsDescriptor(string bucket) => Self.Bucket = bucket;
 
+#pragma warning disable 618
```

Comment:
Can remove pragma directive I think, now that the methods are marked `[Obsolete]`

Reply:
:+1: